Automattic/jetpack
# Boost: Fix stylesheet loading when cloud css is generating

Fixes 62-gh-Automattic/boost-hydra

#### Changes proposed in this Pull Request:

* Check for Cloud CSS generation string and if it exists, do not alter stylesheet loading. This prevents broken stylesheets during the first few frames of page load

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Define constant `JETPACK_BOOST_CLOUD_CSS` as true to enable Cloud Critical CSS.
* Navigate to Jetpack boost dashboard: `wp-admin/admin.php?page=jetpack-boost`
* Open the site home page in a new browser tab and then open Inspect elements
* Open the first tab with Jetpack boost dashboard and enable Cloud CSS or click regenerate.
* Immediately open the second tab and refresh the page.
* Confirm that the Critical CSS does not exist under `head` tag.
